lib/parser-stats.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const promisify = require('bluebird').promisify;
+const Table = require('cli-table');
+const glob = promisify(require('glob').glob);;
+const filesize = require('filesize');
+const percent = require('calc-percent');
+const fs = require('fs');
+
+
+var table = new Table({
+	head: ['files', 'original size', 'optimized size', 'saving'],
+});
+
+function stats(config) {
+	return Promise.all(
+			Object.keys(config.parsers).map(function(name){
```

Comment:
I would still prefer iterating over filetypes:
```
const fileTypes = {
  css: '**/*.css',
  images: '...?'
}
```
as users can edit the patterns and multiple parsers can have overlapping patterns (think cssmin and uncss).